### ISSUE TITLE
Documentation update

### ISSUE THREAD

**Issue by TheRealShadow** [mschnitzer/ark-survival-ascended-linux-container-image#5]
After attempting the install I had multiple errors ongoing. Maybe good to have these known in the install manual.

pip install requests==2.28.1 | everything higher will crash on the error: TypeError: request() got an unexpected keyword argument 'chunked'.

requests can not be higher in combination with docker.

Also python3 urllib3 must be installed.

Might be a fault at my side, but a couple "good to knows" always might help!

**Comment by mschnitzer** [comment]
This error occurs while using docker on your OS or does it occur inside of the container? What OS are you running?

**Comment by DarkVentHead** [comment]
It occurs inside the asa-server container in logs. I am running Debian Bookworm (12?)

**Comment by TheRealShadow** [comment]
This error occurred while trying to start the server using docker. It seems to be a sub-service of docker itself. My server is on Ubuntu 22.04

**Comment by mschnitzer** [comment]
I tested this setup on a fresh installation of Ubuntu 22.04.3 LTS on Saturday and didn't encounter any issues. But I reduced the amount of packages that I recommended to install for a setup on Ubuntu 22.04, as I learned my earlier recommendation was not the best one.

It might be that there's some other software installed on your system that pulled different Python versions as dependencies or something similar to that.
